pkgs/by-name/pa/parse-changelog/package.nix

```diff
@@ -0,0 +1,44 @@
+{
+  fetchCrate,
+  lib,
+  parse-changelog,
+  rustPlatform,
+  testers,
+  nix-update-script,
+}:
+
+let
+  pname = "parse-changelog";
+  version = "0.6.11";
+in
+
+rustPlatform.buildRustPackage {
+  inherit pname version;
+
+  # Fetch from crates.io because upstream does not publish a `Cargo.lock`.
+  src = fetchCrate {
+    inherit pname version;
+    hash = "sha256-B+uR+cZWsZ9ard1zQx5irfBAIvBfd2Vs11qT/MAlTKw=";
+  };
+
+  cargoHash = "sha256-VfwUga2cmRIFB2wsejqpzgCr/gXzNXwF2tjcyP06KFc=";
+
+  passthru = {
+    tests.version = testers.testVersion { package = parse-changelog; };
+    updateScript = nix-update-script { };
+  };
```

Comment:
```suggestion
  nativeInstallCheckInputs = [ versionCheckHook ];
  versionCheckProgramArg = "--version";
  doInstallCheck = true;

  passthru.updateScript = nix-update-script { };
```

Reply:
Nice, I have not seen `versionCheckHook` before 👍